Docker.md

```diff
@@ -0,0 +1,264 @@
+# Docker for .NET
+
+## Docker overview
+
+Motivation behind using docker is having a reproducible environment for application development and deployment.
+
+Instead of installing libraries, tools, configuration, etc. - everywhere the application runs - a [docker image](#docker-image) which contains everything application needs to run (tools, add configuration files/resources, libraries, etc.). This image is then consistent between development machine, cloud host, VPS, etc. [Docker container](#docker-container) is self-contained and isolated - other applications or different versions of some tool running on host won't affect container execution. Likewise once something is added to an image, even if the remote source is gone, the image contains all the files it needs to run.
+
+All cloud providers support running docker images and it usually it simplifies the development/deployment/testing cycle (fewer "Works on my machine !" scenarios).
+
+Docker images add some overhead :
```

Comment:
FROM:
Docker images add some overhead :
TO:
Docker images do add some overhead :